src/ai/backend/manager/idle.py

```diff
@@ -307,24 +305,133 @@ class UtilizationIdleChecker(BaseIdleChecker):
     """
 
     name: ClassVar[str] = "utilization"
-    default_cpu_util_threshold: ClassVar[float] = 30.0
-    default_accelerator_util_threshold: ClassVar[float] = 10.0
-    default_initial_grace_period: ClassVar[float] = 300.0  # allow first 5 minutes to be idle
```

Comment:
grace-period is for delaying session-checker?
I think we may not need the grace-period decision of idle execution based on multiple data points in a window so that early values should not have much impact on average overall. Also, early points get popped out first as time passes.

Reply:
Hm.. I think it's better to keep the initial grace period, if we want to **apply the utilization idle checker for *interactive* sessions**. Otherwise, it may be removed.

Reply:
Added grace period. check_session will continue execution after grace period time.

Reply:
Since the utilization checker does not terminate a session until the length of the collected data exceeds `window-size`. So, I'm not sure `initial-grace-period` is required. But, there may be use cases to set longer `initial-grace-period` with shorter `window-size`, so I don't have an objection to the introduction of the `initial-grace-period`. So, I'll mark this issue as resolved.